roles/ansible_repositories/tasks/redhat.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Install centos-release-ansible29 on CentOS8'
+  yum:
+    name: centos-release-ansible-29
+    state: latest
+    update_cache: true
+  tags:
+    - packages
+  when:
+    - ansible_distribution == 'CentOS'
+    - ansible_distribution_major_version == '8'
+
+- name: 'Install centos-release-ansible29 on Rocky8'
+  yum:
```

Comment:
Since this is targeted at EL8 OS should this use `dnf` ?

Reply:
Fair enough, I've updated it! :)